core/claims/common_test.go

```diff
@@ -14,9 +12,3 @@ func hexStringToKey(s string) merkletree.Hash {
 	copy(keyBytes[:], keyBytesHex[:merkletree.ElemBytesLen])
 	return merkletree.Hash(merkletree.ElemBytes(keyBytes))
 }
-
```

Comment:
Please, don't remove this function and use it where needed!

Reply:
Commented above